astor/code_gen.py

```diff
@@ -566,7 +566,7 @@ def visit_Str(self, node, is_joined=False):
             def recurse(node):
                 for value in node.values:
                     if isinstance(value, ast.Str):
-                        self.write(value.s)
+                        self.result.append(value.s)
```

Comment:
`'\n'` is already special-cased in the `write()` method:

https://github.com/berkerpeksag/astor/blob/d9f6679f7159230db48fb852ad083fb951bca855/astor/code_gen.py#L171-L172

What do you think about fixing this in the `write()` method? (or even in the `newline()` method) Perhaps we could handle the `'\n'` case in the `newline()` method:

```py
if node is None and extra == 0:
    self.result.append('\n')
```

Perhaps we could even add a new `without_indentation` keyword argument to minimize breaking user code?

Reply:
I'm a bit unsure why it was special-cased before - If I remove the special case at all, all the tests pass including the new one. I tried to look up git history but doesn't find valuable info :(

Reply:
Got it now - so I think writing self.newline in the cases where indentation is needed would be more explicit. Tests green again :)